tests/main.rs

```diff
@@ -88,4 +93,39 @@ mod integ_tests {
         .await
         .unwrap()
     }
+
+    // TODO: Currently ignored because starting up the docker image with TLS requires some hoop
+    //  jumping. We should upgrade CI to be able to do that but this was manually run against
+    //  https://github.com/temporalio/customization-samples/tree/master/tls/tls-simple
+    #[tokio::test]
+    #[ignore]
+    async fn tls_test() {
+        let sgo = ServerGatewayOptions {
```

Comment:
We should have test for the different TLS setups in that repo, (can wait for another PR though).